build_package.py

```diff
@@ -163,17 +167,18 @@ def copy_tree(src, dst):
             copy_tree(source, destination)
         else:
             if not os.path.exists(destination) or \
-                    os.stat(src).st_mtime - os.stat(dst).st_mtime > 1:
+                                    os.stat(src).st_mtime - os.stat(
+                                dst).st_mtime > 1:
```

Comment:
Are we sure about this indentation? (Not a major issue, just looks weirder than the original one:) )